tests/tests_launch/test_launch.py

```diff
@@ -179,6 +179,15 @@ def mock_cuda_run_info(monkeypatch):
     )
 
 
+def code_download_func(dst_dir):
+    with open(os.path.join(dst_dir, "train.py"), "w") as f:
```

Comment:
maybe wrap this block with a `try` in case these files aren't found and then throw a LaunchError?

Reply:
Yes you are correct, I think in my scrolling frenzy I missed that this is inside a test.